### ISSUE TITLE
better intra-page navigation

### ISSUE THREAD

**Issue by dberansky** [twosigma/beakerx#4188]
Perhaps my use case is not standard as the content is not strictly sequential.  Instead, each cell is responsible for specific analysis of a particular part of the dataset and then the results are aggregated into one HTML cell at the top.  So there is a report cell, a formatting cell with CSS for the report, initialization cells for Python and Javascript and then one Python and one Javascript cell for each analysis section.  

In other words, the notebook page is getting long and I have constantly scroll up and down to jump from cell to cell depending on which part of the report I'm working.  Some sort of a navigational widget, like a floating table of content, would be very useful.

**Comment by scottdraves** [comment]
yes a table of contents is a good idea.  it has come up a few times, though the only place i can find in the issue tracker is here: https://github.com/twosigma/beaker-notebook/issues/3785
there is a big UI redesign on the horizon and this was mentioned for that.
